src/server/scripts/Commands/cs_player.cpp

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright (C) 2016+     AzerothCore <www.azerothcore.org>, released under GNU GPL v2 license: https://github.com/azerothcore/azerothcore-wotlk/blob/master/LICENSE-GPL2
+ * Copyright (C) 2008-2016 TrinityCore <http://www.trinitycore.org/>
+ * Copyright (C) 2005-2009 MaNGOS <http://getmangos.com/>
+ */
```

Comment:
Since this is a new file, it needs the other copyright text (agpl)

Reply:
ok, done